src/main/scala/io/iohk/ethereum/nodebuilder/NodeBuilder.scala

```diff
@@ -129,8 +129,8 @@ trait ForkResolverBuilder {
   self: BlockchainConfigBuilder =>
 
   lazy val forkResolverOpt =
-    if (blockchainConfig.customGenesisFileOpt.isDefined) None
-    else Some(new ForkResolver.EtcForkResolver(blockchainConfig))
+    if (blockchainConfig.daoForkConfig.isEmpty) None
+    else Some(new ForkResolver.EtcForkResolver(blockchainConfig.daoForkConfig.get))
```

Comment:
we can use `match` to avoid calling `.get` on opt

Reply:
we no longer consider customGenesisFile? What if someone is running custom network with dao-fork version? I think in that case forkResolver should also be None

Reply:
:thinking: I'm not quite sure I understood you. 

Fork resolver was meant to deal with a dao fork like, it shouldn't matter if you are in a private blockchain. If user is running a custom network with dao-fork version it should pass dao fork checks too. Are you thinking a different scenario?

Reply:
Ok, I think it make sense :)